extension-list

```diff
@@ -293,3 +293,4 @@ $IP/skins/erudite/skin.json
 $IP/skins/foreground/skin.json
 $IP/skins/liberty/skin.json
 $IP/skins/pivot/skin.json
+$IP/skins/BlueSky/skin.json
```

Comment:
Should be before "Citizen"

Reply:
Unrelated to this PR.

Reply:
How so?

Reply:
Look at the lines above that. The whole file needs work and it does not seem worth the effort.

Reply:
No everything else is in order.

Reply:
See 7dd66a3.

Reply:
That's not how it should be ordered though.

Reply:
Capital comes before lowercase in this file.

So Z would come before a in this instance.

Reply:
Done, I believe.

Reply:
Yes, thank you!